src/perf_counters.h

```diff
@@ -87,6 +87,9 @@ class BENCHMARK_EXPORT PerfCounterValues {
 // called, to obtain the object, until the object's destructor is called.
 class BENCHMARK_EXPORT PerfCounters final {
  public:
+  // A handy type helper
+  using ResultType = std::vector<std::pair<std::string, double>>;
```

Comment:
is this used anywhere?

Reply:
It has forcibly to be declared by the user (of the class) every time they call `Stop()` since it is passed by reference. It was twice in the perf_counter_gtest.cc tests but for some reason I reverted.